README.md

```diff
@@ -142,18 +119,10 @@ Facilitators:
 - Sara Trapetti ([@SaraTrap](https://github.com/SaraTrap)), SIGHUP
 - Jade Applegate ([@jadeapplegate](https://github.com/jadeapplegate)), Indeed
 
-### Mentoring WG
-
-[Charter](/mentoring/README.md)
-
-Facilitators:
-
-- Nate Waddington ([@nate-double-u](https://github.com/nate-double-u)), CNCF
-- Jay Tihema ([@jaytiaki](https://github.com/jaytiaki)), ii
-- Ihor Dvoretskyi ([@idvoretskyi](https://github.com/idvoretskyi)), CNCF
-
 ### Community Infrastructure Team
```

Comment:
I would like to see us use "initiative" or something less formal. Team implies that it's a structured group lead by specific people. What we have is a request to collaborate on items such as inclusiveness and community infrastructure as part of the contributor growth WG. There isn't a formal leader, group or meeting times so team is a bit misleading.

Reply:
Changed to initiative.